WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -32,7 +32,8 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         ReaderWelcomeBanner.displayIfNeeded(in: tableView)
-        tableView.register(ReaderTopicsCardCell.self, forCellReuseIdentifier: readerCardTopicsIdentifier)
+
+        tableView.register(ReaderTopicsCardCell.defaultNib, forCellReuseIdentifier: readerCardTopicsIdentifier)
```

Comment:
```suggestion
        tableView.register(ReaderTopicsCardCell.defaultNib, forCellReuseIdentifier: ReaderTopicsCardCell.defaultReuseID)
```

[nitpick] By making `ReaderTopicsCardCell` conform to `NibReusable` instead, you get `defaultReuseID` for free!

Reply:
Done in: 616998ecec24851a03b947bb70a219a03766ce49